package.json

```diff
@@ -28,6 +28,7 @@
     "history": "^4.10.1",
     "lodash.get": "^4.4.2",
     "lodash.kebabcase": "^4.1.1",
+    "moment": "^2.29.4",
```

Comment:
can we see if we can use a different lib than moment.js? that is what we us in zero, but they have been recommending for a while that people use a different library. maybe pick something from this page? or pick a couple and present it in the chat room and we can decide? i haven't looked into it yet, so i'm not sure which would be best. 
https://momentjs.com/docs/#/-project-status/recommendations/